Library/Homebrew/.rubocop_todo.yml

```diff
@@ -20,6 +20,7 @@ Style/Documentation:
     - 'utils.rb'
     - 'utils/fork.rb'
     - 'utils/gems.rb'
+    - 'utils/git_repository.rb'
```

Comment:
Can docs be added (as these are public methods) instead of this being added to `todo`?

Reply:
How do I fix this style error?

```
Library/Homebrew/utils/git_repository.rb:4:1: C: Missing top-level module documentation comment.
module Utils
^^^^^^
```

What about `utils/popen` and `utils/fork` (which both extend `Utils`)? Would adding top-level documentation comments cause conflicts?

Reply:
> How do I fix this style error?

Ah, yes, I see. Would need to perhaps define that module in `utils.rb` and document it? I think it's fine to punt on that, then.